### DISCUSSION TITLE
streamlink-how to connect to server?

### DISCUSSION THREAD

**Discussion by jamie-203** [iptv-org/community#245] [Q&A] [answered]
Newbie here sorry for my ignorance!
Trying to connect to 2 hosts provided by @LaneSh4d0w and @keystroke3
I use
streamlink.exe https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=
and 
streamlink.exe https://linkastream.co/headless?url=https://www.youtube.com/c/kenyacitizentv/live

get error no plugin can handle Url
What am I missing??
Many thanx

**Comment by BellezaEmporium** [comment]
Could be because of the fact both of our services are redirecting to the link once Streamlink has it. Basically, it's Streamlink querying another Streamlink-backed server.

**Comment by keystroke3** [comment, marked as answer]
linkastream is facing some development and scaling issues at the moment and can't serve requests. As @LaneSh4d0w said, the services will send a redirect signal to the browser/application to the streaming link. It is there for easier to just plug in the the url directly into whatever service you are using. The link provided can act as a stand-in link for whatever stream you want. You can also use the service as an api to integrate into a script/application. This will return the stream link instead of redirecting to it. To achieve the second behavior, you need to add `&json=true` to the end of the url if you are using linkastream or use `/api?` instead of `/iptv-query?` if you are using query-streamlink.   
Using `query-streamlink` will return a json object in the form:
```json
{
    "quality":"long m3u8 link here",
}
```
`linkastream.co` will return something like this:
```json
{
   "m3ulink":"long m3u8 link here",
}
```

A standard response from all services is still in the works.

**Reply by jamie-203** [reply]
thanx for info

**Reply by keystroke3** [reply]
Please mark question as answered.
